### PR TITLE
[Filters/FocusManager] Manage filter focusing

### PR DESCRIPTION
**Pull request by AndrewMusgrave**
### WHY are these changes introduced?

Fixes - internal issue linked below

### WHAT is this pull request doing?

* Only add a focus item when trapping is true - multiple trap focusing can be on a page, while not active

## 

### How to 🎩

* Build consumer into web & test focus management with the Filters components

### CODE REVIEW

**Comment by AndrewMusgrave** [comment]
Gif include in linked PR

File: UNRELEASED.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Bug fixes
 
 - Fixed alignment of `Page` and `TopBar` so the search aligns to the page. ([#3610](https://github.com/Shopify/polaris-react/pull/3610))
+- Fixed `FocusManager` adding inactive items([#3630](https://github.com/Shopify/polaris-react/pull/3630))
```
Comment (kyledurand) [inline UNRELEASED.md:18]: Can you elaborate on this? It's hard to tell exactly what the fix is here

**Review by kyledurand — APPROVED** [review]
Code looks good just one comment around wording in the changelog

**Review by ashwinnarayanan2001 — APPROVED** [review]

File: UNRELEASED.md (hunk 1 above)
Comment (AndrewMusgrave) [reply UNRELEASED.md:18]:
```suggestion
- Fixed `FocusManager` from tracking inactive items that prevented trap focusing([#3630](https://github.com/Shopify/polaris-react/pull/3630))
```

**Pull request merged**